### PR TITLE
Add Toby which is a YouTube player for the desktop to the open source category

### PR DESCRIPTION
**Pull request by frankhale**
- Toby is located at https://github.com/frankhale/toby

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Some good apps written with Electron.
 - [Yoda](https://github.com/whoisandie/yoda) - Browse and download YouTube videos.
 - [Particle Dev](https://github.com/spark/spark-dev) - Professional IDE for Particle.
 - [Loop Drop](https://github.com/mmckegg/loop-drop-app) - MIDI looper and synth for live electronic music performances.
+- [Toby](https://github.com/frankhale/toby) - A YouTube player for the desktop.
```
Comment (sindresorhus) [inline readme.md]: > YouTube player.

**Comment by arthurvr** [comment]
Needs a rebase too.

**Comment by frankhale** [comment]
Changes have been completed.

File: readme.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Some good apps written with Electron.
 - [Particle Dev](https://github.com/spark/spark-dev) - Professional IDE for Particle.
 - [Loop Drop](https://github.com/mmckegg/loop-drop-app) - MIDI looper and synth for live electronic music performances.
 - [Fireball](https://github.com/fireball-x/fireball) - Hackable game editor for creating mobile and HTML5 games.
-
+- [Toby](https://github.com/frankhale/toby) - A YouTube player.
```
Comment (sindresorhus) [inline readme.md]: Not what I asked for.

```
- [Toby](https://github.com/frankhale/toby) - YouTube player.
```

**Comment by frankhale** [comment]
Changes made.

**Comment by frankhale** [comment]
Thank you!

**Pull request closed without merging**